webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -8778,6 +8788,8 @@ var CURRENT_GROUP_DATA={};
 
                 var displayName = mckMessageLayout.getTabDisplayName(contact.contactId, isGroup);
                 var notificationsound = mckNotificationTone;
+                clearTimeout(MCK_CHAT_POPUP_TEMPLATE_TIMER);
```

Comment:
@CaptainT33mo MCK_CHAT_POPUP_TEMPLATE_TIMER is not required here?

Reply:
@devashishmamgain removed.